.hound.yml

```diff
@@ -0,0 +1,4 @@
+eslint:
+  enabled: true
+  config_file: .eslintrc
```

Comment:
I thought to set `{ "env": { "es6": true } }` automaticatlly configures the enviroment to ES6. @sachungo

Reply:
It's been a while since I worked with houndCI but `"es6": true` didn't work until `ecmaVersion` was added. I don't know if houndCI already fixed the issue. However, we can try it out. If it still make the noise, then `ecmaVersion` will be added